src/core_atmosphere/physics/Makefile

```diff
@@ -165,6 +165,10 @@ mpas_atmphys_driver_sfclayer.o: \
 mpas_atmphys_finalize.o: \
 	mpas_atmphys_lsm_noahmpfinalize.o
 
+mpas_atmphys_lsm_noahmpfinalize.o : \
+	mpas_atmphys_vars.o \
+	core_physics_noahmp
```

Comment:
I'd suggest moving this dependency block below that for `mpas_atmphys_lsm_noahmpinit.o`. Also, we're guaranteed that `core_physics_noahmp` will be built before any of the object files in `$(OBJS)`, so I would suggest removing `core_physics_noahmp` for consistency with other targets in this directory that also depend on Noah-MP.

Reply:
I missed that `core_physics_noahmp` is added as a dependency in later commits. 
Making bug fix changes based on the the originating commit leaves it open to rediscovering or fighting against issues solved in later commits.